bunsen/bunsen-core/src/main/java/com/cerner/bunsen/definitions/StructureDefinitions.java

```diff
@@ -94,30 +108,27 @@ public static StructureDefinitions create(FhirContext context) {
 
     Class structureDefinitionsClass;
 
-    if (FhirVersionEnum.DSTU3.equals(context.getVersion().getVersion())) {
-
+    FhirVersionEnum versionEnum = context.getVersion().getVersion();
+    if (FhirVersionEnum.DSTU3.equals(versionEnum) || FhirVersionEnum.R4.equals(versionEnum)) {
+      String className = STU3_DEFINITIONS_CLASS;
+      if (FhirVersionEnum.R4.equals(versionEnum)) {
```

Comment:
can we re-write this so we have an if/elif/else that selects the class name and removes the need for a nested if statement?

Reply:
Done.